delivery_carrier_deposit/wizard/deposit.py

```diff
@@ -40,23 +40,33 @@ def _get_carrier_type_selection(self):
         required=True, help="Carrier type (combines several delivery "
         "methods). Make sure that the option 'Deposit Slip' is checked on "
         "the delivery methods that have this carrier type.")
+    picking_type_id = fields.Many2one(
+        comodel_name='stock.picking.type',
+        string='Picking Type',
+        required=True)
```

Comment:
Maybe a default value with the first picking_type_id of out type in the company could be benefit here